app/components/wikia-ui-components/error-info.js

```diff
@@ -10,5 +10,5 @@
 import Component from '@ember/component';
 
 export default Component.extend({
-	classNames: ['error-info', 'wds-font-size-s']
+	classNames: ['error-info', 'wds-font-size-s', 'wds-line-height-l']
```

Comment:
while we're here
tagName: ''
and add div to the template with all these clases